src/Services/activityService.js

```diff
@@ -1,14 +1,18 @@
 import axios from 'axios';
 
-const URL = 'http://ongapi.alkemy.org/public/api';
+const URL = process.env.REACT_APP_ACTIVITY_URL;
 
+export const getActivityId = async (id) => {
```

Comment:
it should be getActivity or getActivityById because getActivityId makes to think that you are getting the id of an activity

Reply:
Perfect, already change it